### ISSUE TITLE
Hyperparameters Lesson - Learning Rate

### ISSUE THREAD

**Issue by sahiljuneja** [udacity/dlnd-issue-reports#283]
This was a very good lesson! 

I had one point that I would like to discuss. You add some specific cases in the video -
 
- Validation error decreasing gradually
- Validation error decreasing in a zig-zag pattern
- Validation error decreasing too slowly
- Validation error increasing

However, shouldn't there be one more case -

- Validation error oscillating but overall decreasing. 

An example of the above case is something like the error value as training progresses being - 6.5, 5.4, 5.7, 4.9, 4.5, 4.75

So the value increases and decreases but globally tends to decrease. This is similar to the 2nd quiz in the Quiz lesson for Learning Rate as well. 

I have noticed the above behavior in the 3rd and 4th projects if learning rate is a bit high. So I was wondering if this should be pointed out or not.

Let me know what you guys think and if the above was clear.

**Comment by sahiljuneja** [comment]
cc - @jalammar
